Makefile

```diff
@@ -14,6 +14,8 @@
 
 VERSION ?= $(shell cat VERSION)
 OPENSHIFT_VERSION ?= $(shell cat OPENSHIFT_VERSION)
+OPENSHIFT_VERSION ?= $(shell cat OPENSHIFT_VERSION)
+ISO_VERSION ?= $(shell cat ISO_VERSION)
```

Comment:
Can we inline this version files? I find we have a bit of a proliferation of version files here.